### ISSUE TITLE
HTML Prints 0 pages

### ISSUE THREAD

**Issue by tresf** [qzind/tray#280]
Using QZ Tray 2.1, printing to an Epson TM88V printer using Pixel printing mode receive the message `Document will be printed across 0 pages`.

* The Epson is configured with a page size of 80mm x 3276mm (3.15in x 129in)
* The same issue does NOT occur if testing against XPS document writer configured at (8.5in x 11in)

**Edit:** Updated logs for correct printer.
```diff
[DEBUG] 2018-02-24 13:22:45,896 @ qz.printer.PrintServiceMatcher:?
	Found 7 printers
[DEBUG] 2018-02-24 13:22:45,898 @ qz.printer.PrintServiceMatcher:?
	Found match: EPSON TM-T88V Receipt5
[DEBUG] 2018-02-24 13:22:45,901 @ qz.utils.PrintingUtilities:?
	Found Resolution: 18000x18000 dphi
[DEBUG] 2018-02-24 13:22:46,243 @ qz.printer.action.PrintHTML:?
	Parsed 1 html records
[TRACE] 2018-02-24 13:22:46,476 @ qz.printer.action.PrintHTML:?
	 Collation = COLLATED
 Copies = 1
 Sides = ONE_SIDED
 JobName = QZ Tray HTML Print
 Page ranges = null
 Print color = MONOCHROME
 Print quality = NORMAL
 Print resolution = Feed res=180dpi. Cross Feed res=180dpi.
 Paper source = Paper source : Automatic
 Page layout = Paper=Paper: Default size=226.368x9285.984 POINT Orient=PORTRAIT leftMargin=0.0 rightMargin=0.0 topMargin=0.0 bottomMargin=0.0
[TRACE] 2018-02-24 13:22:46,484 @ qz.printer.action.WebApp:?
	Waiting on print..
[TRACE] 2018-02-24 13:22:46,529 @ qz.printer.action.WebApp:?
	Setting starting size 301.82399999999996:0.0
[TRACE] 2018-02-24 13:22:46,532 @ qz.printer.action.WebApp:?
	Done: 100.0 > -1.0
[TRACE] 2018-02-24 13:22:46,534 @ qz.printer.action.WebApp:?
	New state: SUCCEEDED > READY
[TRACE] 2018-02-24 13:22:46,538 @ qz.printer.action.WebApp:?
	Done: -1.0 > 0.0
[TRACE] 2018-02-24 13:22:46,542 @ qz.printer.action.WebApp:?
	New state: READY > SCHEDULED
[TRACE] 2018-02-24 13:22:46,545 @ qz.printer.action.WebApp:?
	New state: SCHEDULED > RUNNING
[TRACE] 2018-02-24 13:22:46,582 @ qz.printer.action.WebApp:?
	Done: 0.0 > 50.0
[TRACE] 2018-02-24 13:22:46,604 @ qz.printer.action.WebApp:?
	Done: 50.0 > 100.0
[TRACE] 2018-02-24 13:22:46,606 @ qz.printer.action.WebApp:?
	New state: RUNNING > SUCCEEDED
[TRACE] 2018-02-24 13:22:46,627 @ qz.printer.action.WebApp:?
	Setting HTML page height to 432.0
[DEBUG] 2018-02-24 13:22:46,796 @ qz.printer.action.WebApp:?
	Paper area: 0,0:226,9286
[DEBUG] 2018-02-24 13:22:46,798 @ qz.printer.action.WebApp:?
-	Document will be printed across 0 pages
[INFO] 2018-02-24 13:22:47,490 @ qz.utils.PrintingUtilities:?
	Printing complete
[TRACE] 2018-02-24 13:22:47,497 @ qz.utils.PrintingUtilities:?
	Returning processor back to pool
```

**Comment by bberenz** [comment]
Problem with the large height dimension dwarfing the document height. Causing the vertical span check to sit too close to 0.
Fix under ac4ef6ce2b57d5426d63ea731f0fd947ba15d648

**Comment by tresf** [comment]
Closed via ac4ef6c.
